ui/app/AppLayouts/Wallet/controls/SortOrderComboBox.qml

```diff
@@ -241,8 +228,7 @@ ComboBox {
         verticalPadding: isSeparator ? 2 : 5
         spacing: root.spacing
         font: root.font
-        text: root.textRole ? (Array.isArray(root.model) ? modelData[root.textRole] : model[root.textRole])
-                            : modelData
+        text: root.textRole ? modelData[root.textRole] : modelData
```

Comment:
`model` is not available since we only publish `required property var modelData`